sys-apps/bleachbit/bleachbit-2.0.ebuild

```diff
@@ -19,45 +15,33 @@ SRC_URI="https://github.com/${PN}/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
 LICENSE="GPL-3"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
-IUSE="+gtk nls"
+IUSE="+gtk"
 
 RDEPEND="gtk? ( dev-python/pygtk:2[$PYTHON_USEDEP] )"
 
 DEPEND="${RDEPEND}
-	nls? ( sys-devel/gettext )"
+	sys-devel/gettext"
 
 python_prepare_all() {
-	rem_locale() {
-		rm "po/${1}.po" &>/dev/null
-		if [[ $? -ne 0 ]]; then
-			eerror "Removing of ${1}.po failed"
-			die
-		fi
-	}
-
-	l10n_find_plocales_changes po "" ".po"
-	l10n_for_each_disabled_locale_do rem_locale
-
 	# choose correct Python implementation, bug #465254
 	sed -i 's/python/$(PYTHON)/g' po/Makefile || die
 
 	distutils-r1_python_prepare_all
 }
 
 python_compile_all() {
-	use nls && emake -C po
+	emake -C po
 }
 
 python_install_all() {
 	distutils-r1_python_install_all
-	use nls && emake -C po DESTDIR="${D}" install
+	emake -C po DESTDIR="${D}" install
 
 	# https://bugs.gentoo.org/388999
-	insinto /usr/share/${PN}/cleaners
+	insinto /usr/share/bleachbit/cleaners
 	doins cleaners/*.xml
 
-	newbin ${PN}.py ${PN}
-	python_replicate_script "${D}/usr/bin/${PN}"
+	python_newscript ${PN}.py ${PN}
```

Comment:
Err, no. This needs to go into `python_install`, so that it's run for all implementations.

https://wiki.gentoo.org/wiki/Project:Python/distutils-r1#Phase_functions